app/assets/javascripts/discourse/app/components/topic-timeline/container.js

```diff
@@ -287,8 +287,10 @@ export default class TopicTimelineScrollArea extends Component {
   }
 
   @bind
-  dragMove(e) {
-    this.updatePercentage(e);
+  dragMove(event) {
```

Comment:
This is the only other use of "draggable" class. Moved stopPropagation and preventDefault here to keep original behaviour